### PR TITLE
Update draft-amend-tsvwg-multipath-dccp.mkd

### PR DESCRIPTION
**Pull request by dirkhugo**
real update proposal

### CODE REVIEW

**Review by markusa — APPROVED** [review]

**Review by markusa — APPROVED** [review]

File: draft-amend-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -154,7 +197,7 @@ does not provide built-in support for those multiple application subflows.
 In the following, use of the term subflow will refer to physical
 separate DCCP subflows transmitted via different paths, but not to
 application subflows. Application subflows are differing content-wise by
-source and destination application as e.g. enabled by Service Codes
+source and destination application as, e.g., enabled by Service Codes
```
Comment (shjohnson-bt) [inline draft-amend-tsvwg-multipath-dccp.mkd:200]: I don't fully understand this point, but would it read better like this:
"Application subflows are differing content-wise by source and destination application as, for example, enabled by Service Codes introduced to DCCP in {{RFC5595}}, and can be multiplexed over a single DCCP connection."

i.e. I'm not sure if the "can be multiplexed" part relates to the example, or the start of the sentence. (Note the comma after the rfc5595)

File: draft-amend-tsvwg-multipath-dccp.mkd [hunk 2]
```diff
@@ -283,20 +326,20 @@ congestion control.
 | Session handover             | yes                   | yes                |
 | Path aggregation             | yes                   | yes                |
 | Robust session establishment | no                    | yes                |
-| Data reassembly              | yes                   | optional / modular |
+| Data re-assembly             | yes                   | optional / modular |
```
Comment (shjohnson-bt) [inline draft-amend-tsvwg-multipath-dccp.mkd:329]: Indifferent on this - some texts use "reassembly" whilst others use "re-assembly". Important thing is being self consistent

File: draft-amend-tsvwg-multipath-dccp.mkd [hunk 3]
```diff
@@ -399,12 +443,12 @@ considerations for:
 
 
 
-* The minimum PMTU of the individual paths must be selected to
-  announce to the application.  Changes of individual path PMTUs
-  must be re-announced to the application if they are lower than the
-  current announced PMTU.
+* The minimum PMTU of the individual paths must be selected to be
+  announced to the application.  Changes of individual path PMTUs
```
Comment (shjohnson-bt) [inline draft-amend-tsvwg-multipath-dccp.mkd:447]: Would this be better?
"The minimum PMTU of the individual paths must be announced to the application"

File: draft-amend-tsvwg-multipath-dccp.mkd [hunk 4]
```diff
@@ -618,7 +662,12 @@ Smooth RTT (=3)
 {: vspace='0'}
 
 Age (=4)
-: [TBD]
+: The Age parameter is a 4-byte value which is set to the age or  
+  time stamp when the Ack for the datagram was received in case of
```
Comment (shjohnson-bt) [inline draft-amend-tsvwg-multipath-dccp.mkd:666]: "timestamp" is used elsewhere in the doc

File: draft-amend-tsvwg-multipath-dccp.mkd [hunk 5]
```diff
@@ -885,6 +934,12 @@ and use of two additional port numbers out of the UDP port number space, but
 would require zero overhead afterwards.
 
 
+# Implementation
+
+The approach described above has been implemented in open source at different testbeds and a new scheduling algorithm has been extensively tested. Also
```
Comment (shjohnson-bt) [inline draft-amend-tsvwg-multipath-dccp.mkd:939]: "across" different testbeds?

File: draft-amend-tsvwg-multipath-dccp.mkd [hunk 6]
```diff
@@ -899,6 +954,66 @@ unmodified.
 
 # IANA Considerations
 
+This document defines one new value to DCCP feature list and one new  
+DCCP Option with ten corresponding Subtypes as following.
```
Comment (shjohnson-bt) [inline draft-amend-tsvwg-multipath-dccp.mkd:958]: as "follows"

File: draft-amend-tsvwg-multipath-dccp.mkd (hunk 2 above)
Comment (markusa) [reply draft-amend-tsvwg-multipath-dccp.mkd:329]: Yep, and I think in our document it is consistent with "re-assembly". At least I checked again the whole document for some legacy "reassembly"

File: draft-amend-tsvwg-multipath-dccp.mkd [hunk 7]
```diff
@@ -97,10 +98,47 @@ informative:
...
+      -
+         ins: M. Pieska
+         name: Marcus Pieska
+      -
+         ins: A. Kassler
+         name: Andreas Kassler
+      -
+         ins: A. Brunstrom
+         name: Anna Brunstrom
+    date: '2021-10-14'
```
[the remark this answers is not in the document]
Comment (markusa) [reply draft-amend-tsvwg-multipath-dccp.mkd:131]: You made my day. Very good observed. I changed it to the correct year in 2019

File: draft-amend-tsvwg-multipath-dccp.mkd (hunk 1 above)
Comment (markusa) [reply draft-amend-tsvwg-multipath-dccp.mkd:200]: @dirkhugo can you check?

File: draft-amend-tsvwg-multipath-dccp.mkd (hunk 3 above)
Comment (markusa) [reply draft-amend-tsvwg-multipath-dccp.mkd:447]: Incorporated

File: draft-amend-tsvwg-multipath-dccp.mkd (hunk 4 above)
Comment (markusa) [reply draft-amend-tsvwg-multipath-dccp.mkd:666]: I guess you proposed to remove the whitespace? Done

File: draft-amend-tsvwg-multipath-dccp.mkd (hunk 5 above)
Comment (markusa) [reply draft-amend-tsvwg-multipath-dccp.mkd:939]: Done

File: draft-amend-tsvwg-multipath-dccp.mkd (hunk 1 above)
Comment (dirkhugo) [reply draft-amend-tsvwg-multipath-dccp.mkd:200]: 'can be multiplexed' relates to 'application subflows' - thus should relate to the start of the sentence ... I think adding 'port' makes it clearer:
"Application subflows are differing content-wise by source and destination port per application as, for example, enabled by Service Codes introduced to DCCP in {{RFC5595}}, and those subflows can be multiplexed over a single DCCP connection." - thanks!

Comment (markusa) [reply draft-amend-tsvwg-multipath-dccp.mkd]: Taken in b2a1befb205368fe8accf283406dd98bc544170e

**Pull request merged**